blazor/scheduler/resources.md

```diff
@@ -1395,6 +1417,7 @@ It is possible to display tooltip over the resource headers showing the resource
     }
 }
 ```
+![Blazor Scheduler tooltip for resource headers](images/blazor-schedule-tooltip.png)
```

Comment:
The respective image is partially hidden. provide the proper image

Reply:
Changes are made and updated